src/utils.cpp

```diff
@@ -5819,6 +5835,65 @@ struct obj_data *make_new_finished_program(int part_type, int mpcp, int rating=0
   return prog;
 }
 
+struct obj_data *make_otaku_deck(struct char_data *ch) {
+  struct obj_data *asist = find_cyberware(ch, CYB_ASIST);
+  struct obj_data *new_deck = read_object(OBJ_CUSTOM_CYBERDECK_SHELL, VIRTUAL, OBJ_LOAD_REASON_OTAKU_BS);
+
```

Comment:
Please add safeguard checks for `if (!ch || !asist | !new_deck) { log something, return }` here. I never think I'll hit them when coding, but every time one of those safeguard log messages pop, I know they prevented a crash.

Reply:
You can generally assume `read_object()` will never fail, so you can omit that if you want.